data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1169 @@ event "pacified ka'sei"
+			`	Dusk halts mid phrase, looking behind you with an awestruck gaze. The sky has again turned black, and a few dim lines are already starting to appear in the distance. The whole team jumps to the windows, watching as the light show begins.`
+			`	While stars and galaxies zoom in the night sky, two Remnant keep running around the lab, turning on various instruments. However, the rest of them remain with their faces stuck to the glass with you. The lights start converging toward a single point, forming a large, but rapidly shrinking mass in the background.`
+			`	The stars begin to slow down before settling into their final positions. After all the time you have spent here, it isn't difficult to recognize that the sky has changed again. Next to you, the rest of the team is already analyzing the data gathered; Dusk seems to be writing down something from a chronometer.`
+			choice
+				`	"Are we back in the Milky Way?"`
+				`	"How long was that?"`
+					goto time
+				`	"Where are we now?"`
+			`	"I cannot say yet," Dusk replies hurriedly. "Quick, put your EVA suit back on. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
+				goto depart
+
+			label time
+			`	"Around four minutes," Dusk replies hurriedly. "But we will talk about that another time. We need to depart as soon as possible, all of us together. I do not know how much time we have." You hastily connect your helmet back to the life support and follow them out of the airlock, trying to keep up as best as you can.`
+			label depart
+			`	It takes only a few minutes to launch the <ship>, the six of you standing in the cockpit with trepidation. "We will take care of the necessary analysis while you bring us to orbit. Hopefully we have been sent to the Milky Way, and not to yet another unreachable galaxy."`
+				launch
+	on enter
+		conversation "end of Patir 2 - on enter"
+
+	on complete
+		payment 1570000
+		clear "remnant: dusk busy"
+		clear "remnant: dusk busy"
+		event "patir mystery 2 timer" 45
+		set "initial patir exploration"
+		conversation "end of Patir 2 - on complete"
+
+event "Patir returns"
+	system Patir
+		pos -38 553
+		haze _menu/haze-red
+		remove hazard "Beyond Patir Shield Eraser" 1
+		add hazard "Ember Waste Base Heat" 100
+		add hazard "Ember Waste Base Storm" 9000
+
+mission "Ka'het: Patir Mystery 13A"
```

Comment:
Can this be listed before 13B?
Also, passengers?

Reply:
Swapped